hangman.py

```diff
@@ -1,2 +1,27 @@
 # hangman.py 
     
+# Hangman (Pseudocode)
```

Comment:
This function needs to be pasted into the current grade-11b class file. This is pasted into an empty.